packages/vhd-cli/src/commands/compare.js

```diff
@@ -0,0 +1,81 @@
+import { getSyncedHandler } from '@xen-orchestra/fs'
+import { openVhd, Constants } from 'vhd-lib'
+import { resolve } from 'path'
+import Disposable from 'promise-toolbox/Disposable'
+import { omit } from 'lodash'
```

Comment:
You don't need to load the full `lodash`:

```suggestion
import omit from 'lodash/omit'
```